src/Symfony/Component/AssetMapper/ImportMap/Resolver/PackageResolverInterface.php

```diff
@@ -35,7 +35,7 @@ public function resolvePackages(array $packagesToRequire): array;
      *
      * @param array<string, ImportMapEntry> $importMapEntries
      *
-     * @return array<string, string>
+     * @return array<string, array{content: string, dependencies: string[]}>
```

Comment:
Not ideal... not sure if it merits a new object - as this is entirely internal, unless someone makes their own package resolver.